ci-operator/step-registry/hypershift/aws/conformance-cilium/hypershift-aws-conformance-cilium-workflow.metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"path": "hypershift/aws/conformance-cilium/hypershift-aws-conformance-cilium-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"csrwng",
+			"enxebre",
+			"sjenning",
+			"imain",
+			"davidvossel",
+			"LiangquanLi930",
+			"Patryk-Stefanski",
+			"bryan-cox"
+		],
+		"reviewers": [
+			"csrwng",
+			"enxebre",
+			"sjenning",
+			"imain",
+			"Patryk-Stefanski",
```

Comment:
could you move out Patryk and put me in? (jparrill) He is not in the team anymore.